src/Config.cpp

```diff
@@ -451,17 +457,40 @@ bool getStatusConfig()
 void generateRandomSerial()
 {
     randomSeed(millis());
-    sprintf(serial, "SN-%03x-%03x", (unsigned int)random(4095), (unsigned int)random(4095));
+    serial[0]             = 'S';
```

Comment:
i don't like it - but freeing up more than 1.2k is a compelling reason to start liking it.